src/App.jsx

```diff
@@ -12,6 +13,7 @@ const App = () => {
         <Route path="/login" element={<LoginPage />} />
         <Route path="/signup" element={<SignUpPage />} />
         <Route path="/my-learnings" element={<MyLearningsPage />} />
+        <Route path="/cart" element={<ShopingCart />} />
```

Comment:
We usually create a page in pages folder and add the component to it then import this page here, you could follow the same way.

Reply:
ok i will do this .